CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and p
 
 - Fixed incorrect AWSCore dependency version in podspec ([Issue #190](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/190))
 - Fixed data races in AppSyncMQTTClient that were causing crashes ([Issue #184](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/184))
+- Fix a bug in AWSPerformMutationOperation.cancle, so that a mutation can be deleted from the persistent database when it's canceled ([Issue #187](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/187))
```

Comment:
Suggested reword: "Fixed bug where canceling a mutation properly removes it from the persistent mutation cache. ([Issue..."

Reply:
Done